app/javascript/react/components/molecules/Calendar/HeaderToggle/HeaderToggle.style.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import { H3 } from "../../../Typography";
+
+export const Container = styled.div`
+  display: flex;
+  align-items: center;
+  padding-bottom: 20px;
+`;
+
+export const RotatedIcon = styled.img<{ rotated: boolean }>`
+  margin-right: 10px;
+  transform: ${({ rotated }) => (rotated ? "rotate(180deg)" : "none")};
+  cursor: pointer;
+`;
+
+export const Heading = styled(H3)`
+  font-weight: 700;
+  font-size: 22px;
+  cursor: pointer;
+`;
```

Comment:
Should we export each one separately or maybe `export { Container, RotatedIcon, Heading }`. Just pointing it out cc @evulins

Reply:
sure, I can change it :)